app/src/main/java/com/xzyht/notifyrelay/servers/appslist/AppRepository.kt

```
@@ -0,0 +1,659 @@
+            _apps.value = apps
+
+            // 保存应用信息到数据库并加载图标
+            val appEntities = mutableListOf<AppEntity>()
+            val pm = context.packageManager
+
+            apps.forEach { appInfo ->
+                try {
+                    val packageName = appInfo.packageName
+                    val appName = try {
+                        pm.getApplicationLabel(appInfo).toString()
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)